src/main/java/com/eternalcode/discordapp/experience/ExperienceService.java

```diff
@@ -0,0 +1,43 @@
+package com.eternalcode.discordapp.experience;
+
+public class ExperienceService {
+    private ExperienceRepository experienceRepository;
+
+    public ExperienceService(ExperienceRepository experienceRepository) {
+        this.experienceRepository = experienceRepository;
+    }
+
+    public void addPoints(long userId, double points) {
+        this.experienceRepository.find(userId).whenComplete((experience, expierienceThrowable) -> {
```

Comment:
Takie operacje dają wiele do życzenia ogólnie lepiej robić dedykowane metody dla repo takie jak addPoints żeby móc przeprowadzić tranzakcję, już tam w/e ale później trzeba to poprawić

Reply:
Solved